assets/app/scripts/app.js

```diff
@@ -194,6 +195,14 @@ angular
         templateUrl: 'views/browse/service.html',
         controller: 'ServiceController'
       })
+      .when('/project/:project/browse/storage', {
+        templateUrl: 'views/storage.html',
+        controller: 'StorageController'
+      })
+      .when('/project/:project/browse/persistentvolumeclaims/:pvc', {
```

Comment:
The inspiration for `/project/:project/attachPersistentVolumeClaim` was `/project/:project/createRoute`, but I'm open to change it to what we think fits best. Another idea would be to place it under the resource (pretty long though), like

```
/project/:project/browse/deployments/:deploymentconfig/attach/pvc
```